.github/workflows/bundleTest-thor.yml

```diff
@@ -78,6 +77,44 @@ jobs:
           sudo chown -R $USER:$USER /opt/HPCCSystems
           sudo xmlstarlet ed -L -u 'Environment/Software/ThorCluster/@slavesPerNode' -v 2 -u 'Environment/Software/ThorCluster/@channelsPerSlave' -v 1 /etc/HPCCSystems/environment.xml
 
+      - name: Checkout ecl-bundles repository
+        uses: actions/checkout@v4
+        with:
+          repository: hpcc-systems/ecl-bundles
+
+      - name: Scrap Bundles List
+        shell: "bash"
+        run: | 
+          TABLES_DATA=$( cat README.rst | grep -oE -e "\|[^\|]*\|"  -e "==.*==" | sed 's/|//g' )
+          IFS=$'\n'
+          HEADER_NUM_ECL_BUNDLES=2
+          HEADER_COUNT=0
+          for LINE in $TABLES_DATA
+          do
+            LINE=${LINE# }   #removing space from begining of the line
+            LINE=${LINE%% [^A-Za-z0-9]*} #remove trailing spaces.
+            if [[ ${LINE:0:1} == "=" ]]; then 
+              HEADER_COUNT=$(( HEADER_COUNT + 1 ))
+              continue
+            fi
+            if [[ $HEADER_COUNT -eq $HEADER_NUM_ECL_BUNDLES ]]; then 
+              if [[ ${LINE:0:4} == "http" ]]; then
+                echo -e "Bundle Repo : ${LINE}\n" 
+                BUNDLE=$( basename $LINE )
+                BUNDLES_TO_TEST+=" ${BUNDLE%.git}" 
+              else
+                echo "Bundle Name : ${LINE}" 
+              fi
+            elif [[ $HEADER_COUNT -eq $(( HEADER_NUM_ECL_BUNDLES + 1 )) ]]; then 
+              break
+            fi
+          done
+          BUNDLES_TO_TEST=$(echo $BUNDLES_TO_TEST | sed 's/ /\n/g' | sort -bf | uniq -i )
```

Comment:
The sort command already has a -u or --unique parameter. Is that suitable here?

Reply:
Oh! Yes, the sort command has a -u parameter, it would be suitable here and we can eliminate uniq command.